pages/account/manage/profile.js

```diff
@@ -103,6 +116,26 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
     });
   };
 
+  const deleteProfile = async() => {
+    const res = await fetch(`${BASE_URL}/api/account/manage/profile`, {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ name, bio, tags, layout, isEnabled: false}),
```

Comment:
if doing a delete, I don't think all the other data fields are required

Reply:
Using all the data fields could this be simplified by re-using the `handleSubmit` function?

Reply:
> if doing a delete, I don't think all the other data fields are required

while creating the `updateProfile` object in `pages/api/account/manage/profile.js` we are completely relying on `req.body` data. we can also perform findOne call using _id in API but it doesn't felt appropriate to add a new DB call for this. so i thought it's a good idea to send all data.

Reply:
> Using all the data fields could this be simplified by re-using the `handleSubmit` function?

Are you looking for a approach similar to this ?
```
const deleteProfile = async() => {
   handleSubmit(false);  // false is isEnabled flag value
   return Router.push(`${BASE_URL}/account/profile-deactivated`);
}
```
This approach for some reason is adding a reload to profile page whenever handleSubmit is called.

Reply:
I will have a look